javascript-timers/interval.js

```diff
@@ -0,0 +1,16 @@
+var count = document.querySelector('.countdown-display');
+
+function countFunction() {
+  if (count.textContent === '4') {
+    count.textContent = '3';
+  } else if (count.textContent === '3') {
+    count.textContent = '2';
+  } else if (count.textContent === '2') {
+    count.textContent = '1';
+  } else if (count.textContent === '1') {
```

Comment:
What if the countdown needed to start at 1000? You should try to find a way to do this without a conditional block for each number.